python/tvm/autotvm/task/relay_integration.py

```diff
@@ -98,6 +103,8 @@ def extract_from_multiple_program(funcs, params, ops, target, target_host=None):
         The compilation target
     target_host: tvm.target.Target
         The host compilation target
+    template_key: str
```

Comment:
shall we use op -> template_key dict instead?

Reply:
Agree. We could also consider `ops` argument to something like `ops=((relay.op.nn.conv2d, 'direct'), (relay.op.nn.dense, 'direct_nopack'))` to make it more straightforward.

Reply:
That is good point. I have also considered such design, though I thought it was complicating things... While relay ops are mapped to TOPI op, which implements different templates, the `template_keys` needs to be a dict of TOPI op to keys. I will update this.